internal/ingest/upload.go

```diff
@@ -117,3 +137,61 @@ func (w *goaWrapper) initSIP(
 
 	return nil
 }
+
+func (w *goaWrapper) findOrCreateUser(
+	ctx context.Context,
+	perSvc persistence.Service,
+) (*datatypes.User, error) {
+	claims := auth.UserClaimsFromContext(ctx)
+	if claims == nil {
+		return nil, nil
+	}
+	if claims.ISS == "" {
```

Comment:
I just realized it should be `claims.Iss` after seeing your changes, that was my bad. Feel free to leave it or change it now that we are doing it properly in other fields.